### ISSUE TITLE
WebRTC leak

### ISSUE THREAD

**Issue by Allamaris0** [CheshireCaat/selenium-with-fingerprints#16]
I use:    

```
plugin.useProxy(proxy, {
        detectExternalIP: false,
        changeGeolocation: true,
        changeTimezone: true,
        changeWebRTC: true,
    }).useFingerprint(fingerprint);
```

But it shows IP of a proxy provider.
![stealth](https://github.com/CheshireCaat/selenium-with-fingerprints/assets/51059882/34e23b6f-bf9a-4185-ae4b-7b540057ddfc)

**Comment by bablosoft** [comment]
I can't reproduce issue with following code:

```javascript
const { plugin } = require('selenium-with-fingerprints');

(async () => {
  // Get a fingerprint from the server:
  const fingerprint = await plugin.fetch('KEY', {
    tags: ['Microsoft Windows', 'Chrome'],
  });

  // Apply fingerprint:
  plugin.useProxy('PROXY').useFingerprint(fingerprint);

  // Launch the browser instance:
  const driver = await plugin.launch();

  // The rest of the code is the same as for a standard `selenium` library:
  await driver.get('https://browserleaks.com/ip');

  await new Promise(r => setTimeout(r, 2000000));
  await driver.quit();
})();
```

<img width="1417" alt="NVIDIA_Share_2023-11-03_09-04-42" src="https://github.com/CheshireCaat/selenium-with-fingerprints/assets/19365536/e6c9d0f5-6b3d-4843-ae74-d03c7fa6b436">


Please attach full source and send proxy to mail.to.twaego@gmail.com

**Comment by Allamaris0** [comment]
I check proxies with Dolphin browser and there usually a public IP address is covered, but not always. Simultanously I ran script like yours and the public ip wasn't the same. I also checked with puppeteer-with-fingerprints and there is the same problem. However dolphin doesn't always show the same ip address, so maybe it's the fault of my proxies. What is your proxy provider? Maybe I check with another provider.
